dockerfiles/Dockerfile

```diff
@@ -0,0 +1,119 @@
+FROM debian:buster-slim AS labgrid-base
+
+LABEL maintainer="eha@deif.com"
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY ./ /opt/labgrid/
+
+RUN set -e ;\
+    apt update -q=2 ;\
+    apt install -q=2 --yes --no-install-recommends python3 python3-dev python3-pip python3-setuptools python3-wheel git build-essential libsnappy-dev ;\
+    pip3 install -U pip;\
+    apt clean ;\
+    rm -rf /var/lib/apt/lists/* ;\
+    git clone git://github.com/vishnubob/wait-for-it.git opt/wait-for-it && cd opt/wait-for-it  && git reset --hard 54d1f0bfeb6557adf8a3204455389d0901652242
+
+#
+# Client
+#
+FROM labgrid-base AS labgrid-client
+
+RUN set -e ;\
+    cd /opt/labgrid ;\
+    pip3 install yq ;\
+    pip3 install --no-cache-dir -r requirements.txt ;\
+    python3 setup.py install ;\
+    apt update -q=2 ;\
+    apt install -q=2 --yes --no-install-recommends microcom openssh-client sshpass rsync jq qemu-system; \
+    apt clean ;\
+    rm -rf /var/lib/apt/lists/*
+
+CMD ["/bin/bash"]
+
+#
+# Coordinator
+#
+FROM labgrid-base AS labgrid-coordinator
+
+ENV CROSSBAR_DIR=/opt/crossbar
+
+RUN set -e ;\
+    cd /opt/labgrid ;\
+    pip3 install --no-cache-dir -r crossbar-requirements.txt ;\
+    python3 setup.py install
+
+VOLUME /opt/crossbar
+
+EXPOSE 20408
+
+CMD ["crossbar", "start", "--config", "/opt/labgrid/.crossbar/config.yaml"]
+
+#
+# ser2net
+#
```

Comment:
I see the point as the distro release is quite old. However for maintainability would it be possible to inherit from another distro having a newer release?

Reply:
I need the latest release of ser2net (4.3.3) to fix a bug. 

If we want to stick with Debian, I think the only option would be to move to _testing_ (e.g. "bullseye").

Reply:
I see no strong reason to stick to a certain distro if the image works with latest debian base, alpine, fine by me. 
But I see a burden of maintaining packages and dependencies on our self. So would it be possible to try with a distro?

Reply:
@krevsbech I checked Alpine and Fedora, but unfortunately it looks like no one has a version of ser2net newer that 3.5. I'm not quite sure why that is but regardless we would have to compile it manually with one of them also. I think that makes the only remaining option to switch to bullseye. Would you like to try that or compile it locally?

Reply:
I see, then I agree with you that this is the best solution. I will go ahead and approve your contribution.